reference/uitpas.json

```diff
@@ -3984,6 +4191,124 @@
           }
         ]
       }
+    },
+    "/card-systems": {
+      "get": {
+        "summary": "Search card systems",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/CardSystemsPaginatedCollection"
+                },
+                "examples": {
+                  "Example": {
+                    "value": {
+                      "totalItems": 0,
+                      "member": [
+                        {
+                          "id": 1,
+                          "name": "UiTPAS Dender",
+                          "branding": {
+                            "logo": "https://www.uitpas.be/_nuxt/img/1351557.svg",
+                            "primaryColor": "rgba(0,0,0,1.0)",
+                            "secondaryColor": "rgba(97,166,14,1.0)"
+                          },
+                          "links": {
+                            "website": "https://www.uitpas.be"
+                          },
+                          "cities": [
+                            {
+                              "postalCode": "9300",
+                              "name": "Aalst"
+                            },
+                            {
+                              "postalCode": "9400",
+                              "name": "Ninove"
+                            }
+                          ],
+                          "permanent": true
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request.Possbile error types:\n\n* https://api.publiq.be/probs/uitpas/invalid-city\n\nThe detail property might include more information for the client developer."
```

Comment:
An invalid query parameter is technically a 404 Not Found since query parameters are part of the URL, and the URL is the resource locator. So if the query parameter is invalid the URL is invalid, which is a 404.

Alternatively if you don't want to return a 404, you could also accept the invalid city name but return 0 results in the response?

Reply:
Aside from that this PR looks great so this is the last thing to decide :)